--- a/src_ext/Makefile	2018-08-13 14:00:48.678017556 -0700
+++ b/src_ext/Makefile	2018-08-13 14:44:09.801818656 -0700
@@ -164,7 +164,7 @@ endef

 %.stamp: %.download
 	mkdir -p tmp-$*
-	cd tmp-$* && $(if $(patsubst %.tar.gz,,$(URL_$*)),bunzip2,gunzip) -c ../$(call ARCHIVE_FILE,$*) | tar xf -
+	cd tmp-$* && $(if $(patsubst %.tar.gz,,$(URL_$*)),/bin/bunzip2,/bin/gunzip) -c ../$(call ARCHIVE_FILE,$*) | tar xf -
 	rm -rf $*
 	@for ii in tmp-$*/*; do if [ -d $${ii} ]; then mv $${ii} $*; fi; done; \
 	rm -rf tmp-$*
